moveit_ros/visualization/rviz_plugin_render_tools/src/trajectory_visualization.cpp

```diff
@@ -538,7 +538,10 @@ void TrajectoryVisualization::incomingDisplayTrajectory(const moveit_msgs::Displ
     ROS_WARN("Received a trajectory to display for model '%s' but model '%s' was expected", msg->model_id.c_str(),
              robot_model_->getName().c_str());
 
-  trajectory_message_to_display_.reset();
```

Comment:
I think we can simply drop this line of code. There is no reason to reset the pointer here.

Reply:
Sorry for my late response. Your suggestion is of course even better.

Reply:
Should I change the commit or will you take it over?

Reply:
Please change the PR and validate the proposed solution. Thanks.

Reply:
The proposed solution looks good to me. I have changed the PR accordingly.